docs/sdk/api/getCredentials.md

```diff
@@ -0,0 +1,46 @@
+# `get-credentials`
```

Comment:
Not sure, since it only retrieves them and not actually 'load' them into something... Some other opinions would help.